src/List/Item/item.js

```diff
@@ -26,6 +26,7 @@ export default function Item ({ buttons, className, description, icon, isActive,
       className={`${styles.item} ${isActive ? styles.active : styles.inactive} ${className}`}
       onClick={onClick}
       style={style}
+      as={onClick && 'a'}
```

Comment:
I'm maybe wrong, but doesn't SUI automatically transform into a `a` (or at least add `cursor: pointer`) when onClick is not null?

Reply:
hmm doesn't seem so

Reply:
Ah yes, I mixed up with href.